deps/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ if(OS_WINDOWS)
 endif()
 
 add_subdirectory(blake2)
+add_subdirectory(bpm)
```

Comment:
@tt2468 The current branch point for this PR doesn't have a "shared" directory. I'm going to rebase shortly with master, fix conflicts (if any), then move the BPM stuff to "shared" instead of "deps".